akka-docs/src/main/paradox/typed/actor-lifecycle.md

```diff
@@ -16,7 +16,40 @@ TODO intro
 
 ## Creating Actors
 
-TODO
+Actor hierarchy consists of two kinds of actors – a root actor and its children. While there is only one root actor per
```

Comment:
This is a bit weird, not strictly wrong, but also not quite correct :) The root actor isn't a different kind from the children.  So maybe somehow skip that part but keep the next session that details what things are different for the root actor.

What about:
"An actor can _spawn_ an arbitrary number of child actors, which in turn can _spawn_ their own children, forming a hierarchy of actors. The lifecycle of a child actor is tied to the parent - a child can stop whenever it wants but it can  never outlive its parent."